core/lib/spree/testing_support/preferences.rb

```diff
@@ -29,6 +29,33 @@ def assert_preference_unset(preference)
         find("#preferences_#{preference}")['checked'].should be false
         Spree::Config[preference].should be false
       end
+
+      def stub_spree_preferences(preferences)
```

Comment:
I would love to see a comment explaining why we are doing this, including perhaps a link to #3219.

Reply:
@jacobherrington I added some Rdoc/Yard documentation to the new helper methods in `preferences.rb`